test/cli/autogen-path-prefix/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eu
+
+main/sorbet --silence-dev-message --stop-after namer --autogen-version=4 \
+  -p autogen:autogen.txt -p autogen-msgpack:autogen.msgpack \
+  --remove-path-prefix=test/cli/ \
```

Comment:
As far as I can tell, we don't actually use this flag at Stripe anymore, so I'm slightly confused.

If I understand correctly, the test case that you really care about is that if you invoke Sorbet on a directory like `.` (instead of with a list of files), that the `pathPrefix` is set to `./` implicitly.

Do you want to either repurpose this new test to test that behavior, or to add another test for this implicit `pathPrefix` + directory behavior?

Reply:
So `pathPrefix` is implicitly set to `./` _only_ if you don't also pass in files (see [here](https://github.com/sorbet/sorbet/blob/e4aa5da7a277d8a2c7680260162f3cac29c6da8d/main/options/options.cc#L710-L714)), but since there are files in pay-server that are Ruby files but don't have extensions, we need to be able to pass in both a directory and files and also remove the leading `./`. We don't currently use this flag, but we would need it (or at least something similar) in the future, since the test case you're describing is the existing functionality.

Reply:
Got it, that makes sense. Thanks!